### ISSUE TITLE
Replace Maskrcnn with nvidia pytorch

### ISSUE THREAD

**Issue by bitfort** [mlcommons/training_policies#178]
The current maskrcnn implementation uses a large framework which is difficult to understand, debug, etc. It seems to be much easier if we had a slimmed down version which was simpler.

**Comment by bitfort** [comment]
SWG Notes:

One option is Nvidia's submission. Another option is facebook's implementation. 
One proposal is move away from a framework so it is easier for submitters to pick up and use.

AI(NV): make a forward step on this issue for next week.

**Comment by bitfort** [comment]
SWG Notes:

NV recommends using the facebook pytorch code. 

AI(Peter) Invite fb to attend next week submitters to discuss.

**Comment by nvpaulius** [comment]
PR filed, based on FB's PyTorch code: https://github.com/mlperf/training/pull/240
Will still need some cleanup, the main one is switching to COCO 17 data set (doesn't change the data set, but is much cleaner when it comes to train/val breakdown).

**Comment by bitfort** [comment]
SWG Notes:

AI(Intel) - Peer review convergence (1 or 2 days on a V100) and check code.

**Comment by bitfort** [comment]
SWG Notes:

AI(Google) - code review to look for unexpected changes.

**Comment by keithachorn-intel** [comment]
**Updates on:** AI(Intel) - Peer review convergence (1 or 2 days on a V100) and check code.

With the repaired code*, the total time to train on a 1xV100 system (AWS p3.2xlarge) was 63.9 hr (230222s, 11 epochs).

*The reference Mask R-CNN workload has a few broken code references (in the download scripts and indirectly referenced via the Detectron github repo.  As-is, these references must be updated from within the built container.  The specific references were emailed to the Special Topics Google Group, but can be re-posted here by request if useful.

**Comment by keithachorn-intel** [comment]
> **Updates on:** AI(Intel) - Peer review convergence (1 or 2 days on a V100) and check code.
> 
> With the repaired code*, the total time to train on a 1xV100 system (AWS p3.2xlarge) was 63.9 hr (230222s, 11 epochs).
> 
> *The reference Mask R-CNN workload has a few broken code references (in the download scripts and indirectly referenced via the Detectron github repo. As-is, these references must be updated from within the built container. The specific references were emailed to the Special Topics Google Group, but can be re-posted here by request if useful.

This update referenced the current Reference Workload in the Object Detection dir.  Was the request to test an externally-hosted workload?

**Comment by ddkang** [comment]
@nvpaulius @petermattson @bitfort

**Comment by bitfort** [comment]
SWG Notes:

AI(Google, Intel) - Review PR mlperf/training#240

Our goal is to get the reference updated by next meeting.

**Comment by bitfort** [comment]
SWG Notes:

The PR has been merged. Everyone believes that there is no difference in the way the model works. If a difference is discovered, we intend to accept both equivalence with both the v0.5 reference and the v0.6 reference -- for v0.6 submissions ONLY.

**Comment by petermattson** [comment]
No action required.
